Examples/Google/NaturalLanguageNIO/Sources/main.swift

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright 2019, gRPC Authors All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import Dispatch
+import Foundation
+import SwiftGRPCNIO
+import OAuth2
+import NIO
+import NIOHTTP1
+import NIOSSL
+
+/// Prepare a TLSMode for a general SSL client that supports HTTP/2.
+func makeClientTLS() throws -> GRPCClientConnection.TLSMode {
+  let configuration = TLSConfiguration.forClient(applicationProtocols: ["h2"])
+  let context = try NIOSSLContext(configuration: configuration)
+  return .custom(context)
+}
+
+/// Create a client and return a future to provide its value.
+func makeServiceClient(_ eventLoopGroup: MultiThreadedEventLoopGroup,
+                       address: String,
+                       port: Int)
+  -> EventLoopFuture<Google_Cloud_Language_V1_LanguageServiceService_NIOClient> {
+    let promise = eventLoopGroup.next().makePromise(of: Google_Cloud_Language_V1_LanguageServiceService_NIOClient.self)
+    do {
+      try GRPCClientConnection.start(host: address,
+                                     port: port,
+                                     eventLoopGroup: eventLoopGroup,
+                                     tls: makeClientTLS())
+        .map { client in
```

Comment:
Nit: `cascade` the result of the future into the promise:

```swift
.map { client in
    Google_Cloud_Language_V1_LanguageServiceService_NIOClient(connection: client)
}.cascade(to: promise)
```

Reply:
Updated. Thanks for the guidance!